src/lib/viewers/BaseViewer.js

```diff
@@ -860,9 +861,11 @@ class BaseViewer extends EventEmitter {
         const boxAnnotations = this.options.boxAnnotations || new global.BoxAnnotations(viewerOptions);
         this.annotatorConf = boxAnnotations.determineAnnotator(this.options, this.viewerConfig);
 
-        if (this.annotatorConf) {
```

Comment:
Make sure you leave this check in because `boxAnnotations.determineAnnotator()` can return null